app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/RequestedChanges.scala

```diff
@@ -46,6 +50,9 @@ object RequestedChanges {
       "correspDetails" -> model.correspDetails,
       "organisationDetails" -> model.organisationDetails,
       "mandationStatus" -> model.mandationStatus
-    )
+    ) ++ (appConfig.features.api1365Version() match {
+      case Api1365Latest => Json.obj("commsPreference" -> model.commsPreference)
```

Comment:
Not really part of this PR but there isn't a way to toggle the API version via manage VAT - do we want a small task to add this in? You can only choose 'Latest' for both 1363 and 1365 on the feature switch page

Reply:
If you see value in it then yeah I'm happy for a story to be raised. When I added the feature I wondered if we would ever need to toggle that in our test environments, but as there was no immediate need to do so I left it.